lib/db.c

```diff
@@ -53,7 +53,12 @@ int db_init(const char *path_fmt, ...)
 		return -1;
 
 	va_start(ap, path_fmt);
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-nonliteral"
```

Comment:
This one should be fixed by applying the correct attribute on the function itself